src/features/groups/components/CreateGroupButton/index.tsx

```diff
@@ -9,6 +9,7 @@ import { CreateGroupDrawerContent } from './CreateGroupDrawerContent';
 
 export const CreateGroupButton = () => {
   const [isOpen, setIsOpen] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
This line can be defined in the `CreateGroupDrawerContent.tsx` since they are not used in this file, so you don't need to pass them as props.

Reply:
Thank you for the feedback, I changed that 😎